fluid/PaddleCV/human_pose_estimation/lib/pose_resnet.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2018-present, Baidu, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##############################################################################
+
+"""Functions for building network."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import paddle.fluid as fluid
+
+__all__ = ["ResNet", "ResNet50", "ResNet101", "ResNet152"]
+
+BN_MOMENTUM = 0.1
+
+class ResNet():
+    def __init__(self, layers=50, kps_num=16, test_mode=False):
```

Comment:
Need comments for these arguments. 

And mabye the `BN_MOMENTUM` also can be put in these arguments.

Reply:
Done.